cfa_azure/clients.py

```diff
@@ -33,7 +33,8 @@ def __init__(self, config_path: str):
         self.mount_container_clients = []
         self.pool_parameters = None
         self.timeout = None
-
+        self.save_logs_to_blob = None

```

Comment:
Why a default of `None` instead of `False`?

Reply:
It uses `None` because the parameter takes in the name of the blob container to store the outputs, rather than True or False. I thought about using True/False, but if someone has 5 blob containers mounted to their pool, it wouldn't be obvious which blob container the output should be stored in. I wanted to make it as clear as possible where people want to save their output.